.github/workflows/ci.yml

```diff
@@ -181,6 +182,29 @@ jobs:
         if: ${{ github.ref == 'refs/heads/main'
              || startsWith(github.ref, 'refs/tags/v') }}
 
+  docker:
+    needs: ["build"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: build-web-${{ github.run_number }}
+          path: build/web/
```

Comment:
@SleepySquash that's a half of work. As I've [mentioned before](https://github.com/team113/messenger/pull/30#discussion_r917825658), we need to put here all the releases, so we can download them from web.